Exercises.Tests/CacheTests.cs

```diff
@@ -47,7 +47,8 @@ public class CacheTests  {
             var item4 = new ImageDescription() { InstanceUid = Guid.NewGuid().ToString() };
             cache.PutItem(item4.InstanceUid, item4);
 
-            Assert.IsNull(cache.GetItem(item3.InstanceUid), "least recently used item is not removed from cache");
+            //Vishal - changed this
+            Assert.IsNull(cache.GetItem(item1.InstanceUid), "least recently used item is not removed from cache");
```

Comment:
Line numbers 44 and 45 accesses the item1 and item2. But item3 is not accessed after adding. Hence item3 is the least recently used.
Least recently used means the item which is not used recently. It should be item3

Reply:
Agree